test/prism_regression/integer.parse-tree.exp

```diff
@@ -0,0 +1,3 @@
+Integer {
```

Comment:
@Morriar here are my `.parse-tree.exp` tests :) I'm having trouble getting the float test to work, so I'll handle that in another PR.